packages/auth/src/gql/getBaseAuthSchema.ts

```diff
@@ -55,7 +55,7 @@ export function getBaseAuthSchema<I extends string, S extends string>({
         }),
         resolve(root, args, { session, db }) {
           if (
-            (typeof session?.itemId === 'string' || typeof session.itemId === 'number') &&
+            (typeof session?.itemId === 'string' || typeof session?.itemId === 'number') &&
             typeof session.listKey === 'string'
```

Comment:
Should this be `session?.listKey`?

Line 54 uses `context.session?.listKey`.

Reply:
Checks go from left to right, so in case of no session, session.listKey is not executed.
This condition was not changed.

Reply:
Yes, it would short circuit, but I think for readability I might prefer for it to be `?.` until the type is refined within the next block.
I don't think I'll ask for that change in this pull request though, the actual problem is `session` is effectively `any`.